Resources/Prototypes/_Mono/Shipyard/framework.yml

```
@@ -0,0 +1,45 @@
+- type: vessel
+  id: Framework
+  parent: BaseVessel
+  name: UNJ Framework
+  description: The cheapest hunk of junk this side of the Rogue. Not meant to last.
+  price: 9000
+  category: Small
+  group: Shipyard
+  shuttlePath: /Maps/_Mono/Shuttles/framework.yml
+  guidebookPage: null
+  class:
+  - Civilian
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)